src/Illuminate/Queue/QueueServiceProvider.php

```diff
@@ -65,6 +67,21 @@ protected function configureSerializableClosureUses()
         });
     }
 
+    /**
+     * Configure the once instance to flush after each job.
+     *
+     * @return void
+     */
+    public function configureOnce()
```

Comment:
We currently use `once` in jobs and do not flush it between jobs. Flushing it for everyone here, makes this method somewhat less useful IMO.
Similarly, the `array` cache driver does not get flushed on every job run for example.

Reply:
@taylorotwell He may have a point here. While on the HTTP layer, there is the request and it may be unsafe to share state between requests, on the Queue layer could be OK not to flush the state of `once`. 

If we were not to flush state on jobs, I need to update this https://github.com/laravel/octane/pull/821/files pull request to flush `Once` on Request Handled only.

Reply:
I can get on board with removing this part. Done.